conda_build/main_build.py

```diff
@@ -223,8 +223,9 @@ def execute(args, parser):
             if not isdir(d):
                 makedirs(d)
             update_index(d)
+        index = build.get_build_index(clear_cache=True)
```

Comment:
I think I removed this because pyflakes was complaining about index being set but not used.  I see you use it below.  Is there a reason to have it up here, and not down, immediately under

    if args.skip_existing:

?